CMakeLists.txt

```diff
@@ -34,15 +34,31 @@ set(depot_tools_install_dir ${SOURCE_DIR})
 
 set(WEBRTC_REVISION branch-heads/70)
 
-list(APPEND GN_GEN_ARGS
-  rtc_build_examples=false
-  rtc_build_tools=false
-  rtc_use_x11=false
-  rtc_enable_protobuf=false
-  rtc_use_gtk=false
-  rtc_include_pulse_audio=false
-  rtc_include_tests=false
-)
+if ("$ENV{TARGET_ARCH}" STREQUAL "arm")
```

Comment:
Rather than use a `TARGET_ARCH` environment variable, should we be using `CMAKE_SYSTEM_PROCESSOR`? Or does this have some sort of interaction with

* [`node-pre-gyp`](https://github.com/mapbox/node-pre-gyp)'s install process
* [`node-cmake`](https://github.com/cjntaylor/node-cmake)'s build process

Reply:
I guess `TARGET_ARCH` will work better with `npm i`?

Reply:
I think TARGET_ARCH will work better.
We will have more control over process.